From e957c0522770258b9b6d7eb4a96428fb29e6afdb Mon Sep 17 00:00:00 2001
From: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date: Wed, 10 Aug 2016 16:46:55 +0200
Subject: [PATCH] Fix ret type in read_boolean_file

ret is boolean so the comparison (ret < 0) will always evaluate to false.
Change it to the proper type, int

Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
---
 src/libmaxtouch/sysfs/sysfs_device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libmaxtouch/sysfs/sysfs_device.c b/src/libmaxtouch/sysfs/sysfs_device.c
index 6c01f02..5f562ef 100644
--- a/src/libmaxtouch/sysfs/sysfs_device.c
+++ b/src/libmaxtouch/sysfs/sysfs_device.c
@@ -483,7 +483,7 @@ static int read_boolean_file(struct mxt_device *mxt, char *filename,
 {
   FILE *file;
   char val;
-  bool ret;
+  int ret;
 
   file = fopen(filename, "r");
   if (!file) {
